static_docs/case.md

```diff
@@ -0,0 +1,99 @@
+# Case Statement
+
+In Ruby, the `case` statement provides a clean way to express conditional logic when you need to compare a value against multiple conditions. It's similar to if/else chains but often more readable and concise.
+
+```ruby
+# Basic case statement comparing a value against multiple conditions
+grade = "A"
+
+case grade
+when "A"
+  puts "Excellent!"
+when "B"
+  puts "Good job!"
+when "C"
+  puts "Fair"
+else
+  puts "Need improvement"
+end
+```
+
+The `case` statement can also work with ranges, multiple values, and even custom matching using the `===` operator.
+
+```ruby
+# Case statement with ranges and multiple conditions
+score = 85
+
+case score
+when 90..100
+  puts "A grade"
+when 80..89
+  puts "B grade"
+when 70..79
+  puts "C grade"
+else
+  puts "Need improvement"
+end
+
+# Case with multiple values in a single when clause
+day = "Saturday"
+
+case day
+when "Saturday", "Sunday"
+  puts "Weekend!"
+else
+  puts "Weekday"
+end
+```
+
+## Pattern Matching (Ruby 2.7+)
```

Comment:
No need to mention that it's 2.7+. The Ruby LSP only supports 3.0 or higher and 2.7 has been EOL for ages.